### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by Vishruta-Patil**
- Sign in
- Login
- Logout
- Added flaticon and updated readme file

### CODE REVIEW

File: src/components/Router.js [hunk 1]
```diff
@@ -1,17 +1,48 @@
...
-}
\ No newline at end of file
+  
+  const useAuth = () => {
+    const user_data = sessionStorage.getItem("auth_Token");
+
+    return user_data !== null;
+  };
+
+//   const [authStatus, setAuthStatus] = useState(useAuth())
```
Comment (nerdyspook) [inline src/components/Router.js:19]: You can remove this comment from here.

**Review by nerdyspook — APPROVED** [review]
Well structured, liked how you have written comments for todos, doubts.

File: src/components/quiz/QuizCard.jsx [hunk 2]
```diff
@@ -1,83 +1,133 @@
...
-      setActiveOption(false)
-      console.log("no")
-    }
-    
-    if (item.ans === option) {
-      quizDispatch({type:GET_RESULT, payload: result + 1})
-    } else {
-      quizDispatch({type:GET_RESULT, payload: result})
+  const [timer, setTimer] = useState(30);
+  // const [timerId, setTimerId] = useState("");
```
Comment (Devansu-Yadav) [inline src/components/quiz/QuizCard.jsx:17]: Please remove the comments later on.

**Review by Devansu-Yadav — APPROVED** [review]
- UI looks good.
- Clean CSS code
Looks good to merge

**Pull request merged**